borgmatic/hooks/sqlite.py

```diff
@@ -101,9 +105,9 @@ def restore_database_dump(database_config, log_prefix, location_config, dry_run,
         return
 
     try:
-        execute_command(('rm', database_path), shell=True)
-    except CalledProcessError:  # pragma: no cover
-        logger.info(f'{log_prefix}: Database does not exist at {database_path}, skipping removal')
+        os.remove(database_path)
```

Comment:
IMO this should be mocked out in any test that gets this far, so as to avoid attempting file removal on the filesystem during tests. I realize it's unlikely to delete anything real, but I try to avoid unintended filesystem side-effects in tests. Sorry to be a pain here!

Reply:
No worries! I'm not in a hurry to get this merged 😄. Nice catch though, helps me think more about the side effects of my code.